src/main/java/com/sahil/SplitwiseApp/constants/ApiConstants.java

```diff
@@ -0,0 +1,6 @@
+package com.sahil.SplitwiseApp.constants;
+
+public class ApiConstants {
+    public static final String BASE_URL = "/Splitwise-App";
```

Comment:
kebab case should not have words starting with capital letter, have all words in small case

Reply:
didn't know this, noted.